handbook/operations.md

```diff
@@ -0,0 +1,63 @@
+# Operations
+
+## People Ops
+
+### Hiring a new team member
+
+1. Reach out to Mike and Zach and give them an opportunity to meet the candidate.
+
+2. Research compensation using [Pave](https://www.pave.com), making sure to adjust for geographical factors. *If you're unsure of your findings, ask for help from Mike.*
+
+3. Share the compensation information you've gathered with Eric. Eric will confirm all this information with Mike and document this decision in the [compensation decisions document](https://docs.google.com/document/d/1NQ-IjcOTbyFluCWqsFLMfP4SvnopoXDcX0civ-STS5c/edit) for future reference.
+
+4. After compensation has been determined, you need to create two documents for the candidate, an "exit scenarios" spreadsheet and an "informal offer email" template. If you need these templates, reach out to Eric on slack, and he will provide you links that automatically create copies Change the name of the documents accordingly (e.g., "[candidate's name]'s copy of exit scenarios,") and link to the exit scenarios spreadsheet from the offer email.
+
+5. Prepare the informal offer email. You'll need to fill the following information to the template: 
+   - Candidate's name and email address
+   - Candidate's start date
+   - Candidate's compensation
+   - Candidate's reporting manager
+   - Equity offered to the candidate (make this information a link to the candidate's exit scenarios spreadsheet)
+   - Benefits (determined by the candidate's location)
+
+6. Prepare the exit scenarios spreadsheet. Enter the percentage of equity offered to the candidate, and the spreadsheet will update to reflect this. 
+   
+      >**_Note:_** *Don't play with numbers in the exit scenarios spreadsheet. the revision history is visible to the candidate, and they might misunderstand.*
+
+7. Once both documents are complete, share the offer email draft, exit scenarios copy, and a link to the compensation decision, and send them to Eric for confirmation.
```

Comment:
```suggestion
7. Once both documents are complete, share the offer email draft, exit scenarios copy, and a link to the compensation decision, with Eric for confirmation.
```

Reply:
Should it read like this suggestion, or are the listed documents meant to be shared with someone else before sending them to you?